python/whylogs/api/whylabs/session_manager.py

```diff
@@ -0,0 +1,122 @@
+import logging
+import os
+import uuid
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Optional, Union
+
+from whylabs_client import ApiException
+from whylabs_client.api.sessions_api import (
+    CreateSessionRequest,
+    CreateSessionResponse,
+    SessionsApi,
+)
+from whylabs_client.api_client import ApiClient, Configuration
+
+from whylogs.api.whylabs.auth_file import get_auth_file_path
+from whylogs.api.whylabs.notebook_check import is_notebook
+from whylogs.api.whylabs.variables import Variables
+
+DEFAULT_PATH = os.getenv("WHYLOGS_CONFIG_PATH") or f"{Path.home()}/.whylabs/auth.ini"
+_auth_path = get_auth_file_path(auth_path=Path(DEFAULT_PATH))
+
+DEFAULT_WHYLABS_HOST = "https://api.whylabsapp.com"
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class GuestSession:
+    def __init__(self, session_id: str):
+        self.session_id = session_id
+        Variables.set_variable_to_config_file(key="session_id", value=self.session_id, auth_path=_auth_path)
+
+
+@dataclass
+class UserSession:
+    org_id: str
+    api_key: str
+
+    def __post_init__(self):
+        Variables.set_variable_to_config_file(key="org_id", value=self.org_id, auth_path=_auth_path)
+        Variables.set_variable_to_config_file(key="api_key", value=self.api_key, auth_path=_auth_path)
+
+
+def _create_session_id(user_id: str) -> str:
+    config = Configuration()
+    config.host = os.getenv("WHYLABS_API_ENDPOINT") or DEFAULT_WHYLABS_HOST
+
+    try:
+        client = ApiClient(config)
+        api = SessionsApi(client)
+        response: CreateSessionResponse = api.create_session(CreateSessionRequest(user_id))
+
+        logger.debug(f"Created session {response.id}")
+        return response.id
+    except ApiException as e:
+        logger.error(e)
+        raise e
```

Comment:
This is bad. Nothing is catching the error above. 

We should always fall back to local mode if this call cannot be made and just log out a warning

Reply:
What does local mode mean for the anonymous login? this API is only called when creating a GuestSession.
also, if not with ApiException, what else could I use to catch errors here? I'm using the same pattern indicated on other songbird calls across our examples

Reply:
Got it. I thought this was getting called everytime